tests/composite_model_test.py

```diff
@@ -277,7 +285,137 @@ def test_assembly_model(dpf_server):
     )
 
     timer.add("After getting properties")
+    timer.summary()
+
+
+def test_assembly_model(dpf_server):
```

Comment:
Could we not write the tests for the two versions in one tests with some if clauses? So it would be obvious what are the breaking changes.